### PR TITLE
TDD

### PR DESCRIPTION
**Pull request by drewkhoury**
TDD

### CODE REVIEW

**Comment by drewkhoury** [comment]
https://www.katacoda.com/drewkhoury/courses/intro-to-tdd-with-java/scenario-student-tracking-application

Screen Shot 2021-11-17 at 4 26 59 PM

**Comment by drewkhoury** [comment]
I was using a personal account for the testing/ screenshot above, but have since setup a dedicated account which uses the contino fork and it's all good there too.

KataCoda Profile: https://www.katacoda.com/drewkhoury-contino
Login: Google Account Andrew.khoury@contino.io 
Repo: https://github.com/drewkhoury/katacoda-scenarios-1
Fork of: https://github.com/contino/katacoda-scenarios

File: intro-to-tdd-with-java/scenario-student-tracking-application/assets/java/student-tracking-application/.idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (strongjz) [inline intro-to-tdd-with-java/scenario-student-tracking-application/assets/java/student-tracking-application/.idea/compiler.xml:1]: I dont think the .idea directory is needed, can you remove it.

Comment (drewkhoury) [reply intro-to-tdd-with-java/scenario-student-tracking-application/assets/java/student-tracking-application/.idea/compiler.xml:1]: good point. going to abandon this PR in favour of the TDD module Craig is about to submit ... but have passed this note to him.

**Pull request closed without merging**